main.js

```diff
@@ -51,6 +69,7 @@ define(function (require, exports, module) {
     var _preferences,
         _command,
         _styleTag,
+        _styleInline,
```

Comment:
See `unloadStyle()`. It removes `_styleTag` from the DOM when the extension is disabled (a reason currently known only to Dennis). I would alter that function to do the same for `_styleInline`, but I am unable to make jQuery remove it for whatever reason.